src/framework/audio/qml/MuseScore/Audio/VolumeSlider.qml

```diff
@@ -228,7 +228,7 @@ Slider {
                 ctx.textAlign = "end"
             }
 
-            ctx.clearRect(0, 0, bgCanvas.width, bgCanvas.height)
+            ctx.clearRect(0, 0, bgCanvas.height, bgCanvas.width)
```

Comment:
Could you please explain what is the purpose of this change?

Reply:
It's because we're in a rotated world here, so width and height are reversed. 
(In the lines before this, the `ctx` is rotated.)